src/cdk/drag-drop/directives/drag.ts

```diff
@@ -307,12 +307,14 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
     // their original DOM position, and then they get transferred to the portal.
     afterNextRender(
       () => {
-        this._updateRootElement();
-        this._setupHandlesListener();
+        this._ngZone.runOutsideAngular(() => {
```

Comment:
Maybe it was there before, but I don't think that this actually does anything. The drag&drop module binds all the events outside the zone already.